.github/workflows/build.yml

```diff
@@ -0,0 +1,40 @@
+name: Build
+
+on:
+  push:
+    branches: ["main"]
```

Comment:
Do we want this to run whenever we push to any branch e.g. so we can test a build on CODE?

Reply:
Yep can do! I'll change it to run on any push